instance/models/openedx_instance.py

```diff
@@ -77,6 +77,7 @@ class OpenEdXInstance(
     # Most settings/fields are inherited from mixins
 
     successfully_provisioned = models.BooleanField(default=False)
+    current_database_name = models.CharField(blank=True, max_length=60, default='')
```

Comment:
Please rename the field to `database_name`, and the current `database_name` property to `generate_database_name()` (and make it a regular method instead of a property).